Ghost_in_the_Shell/client_folder_react_app/ethisim/src/shared/mockScenarioData.js

```diff
@@ -0,0 +1,245 @@
+//Mock Version History
+const mockEventHistory = [
+    {
+        component: 'Event',
+        title: 'Marius Minea Version Control Introduction',
+        introduction:
+            '<p><span style="font-size: 28px;"><em><u><strong>VERSION CONTROL WORKS #1</strong></u></em></span><em></em><br></p>',
+        date: new Date().getTime(),
+        author: 'Marius Minea',
+        id: 1,
+    },
+    {
+        component: 'Event',
+        title: 'David Fisher Version Control Introduction',
+        introduction:
+            '<p><span style="font-size: 28px;"><em><u><strong>VERSION CONTROL WORKS #2</strong></u></em></span><em></em><br></p>',
+        date: new Date().getTime() - 300000,
```

Comment:
Nothing wrong here, just be very careful with the JS date library, it can be really un-intuitive sometimes. For example days in a month are 1 indexed but months are 0 indexed. (jan 1 is month: 0, day: 1)

Reply:
Good to know